docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>This example is <strong>super tricky</strong>! It shows that the dimension names get attached to a certain shape ("colors"-&gt;2, "group"-&gt;4), and then a previously defined dimension can be used on a completely independent RV, in order to make that parameter have the desired batch dimensions!</p><p>we need to talk about this in more detail</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>Simplified the example</p>